packages/atomic/src/components/ipx/atomic-ipx-button/atomic-ipx-button.tsx

```diff
@@ -38,14 +43,23 @@ export class AtomicIPXButton {
         onClick={() => this.onClick()}
       >
         <span part="button-icon">
-          <atomic-icon
-            part="ipx-close-icon"
-            icon={this.getIcon(CloseIcon)}
-          ></atomic-icon>
-          <atomic-icon
-            part="ipx-search-icon"
-            icon={this.getIcon(SearchIcon)}
-          ></atomic-icon>
+          {this.icon ? (
+            <atomic-icon
+              part="ipx-custom-icon"
+              icon={this.getIcon(this.icon)}
+            ></atomic-icon>
+          ) : (
+            [
+              <atomic-icon
+                part="ipx-close-icon"
+                icon={this.getIcon(CloseIcon)}
+              ></atomic-icon>,
+              <atomic-icon
+                part="ipx-search-icon"
+                icon={this.getIcon(SearchIcon)}
+              ></atomic-icon>,
+            ]
```

Comment:
We generally need to add keys to children of an array when we render an array, I'm not sure what are the exceptions.

For this reason, when I want to render multiple items like this, I prefer to wrap them in a `<Fragment>`.

Reply:
I removed the condition.